client/my-sites/email/email-providers-stacked-comparison/provider-cards/google-workspace-card.tsx

```diff
@@ -203,26 +200,31 @@ const GoogleWorkspaceCard: FunctionComponent< EmailProvidersStackedCardProps > =
 
 	googleWorkspace.onExpandedChange = onExpandedChange;
 	googleWorkspace.formFields = (
-		<FormFieldset className="google-workspace-card__form-fieldset">
-			<GSuiteNewUserList
-				extraValidation={ identityMap }
-				domains={ domainList }
-				onUsersChange={ setGoogleUsers }
-				selectedDomainName={ selectedDomainName }
-				users={ googleUsers }
-				onReturnKeyPress={ onGoogleFormReturnKeyPress }
-				showAddAnotherMailboxButton={ false }
+		<EmailProviderGenericForm
+			extraValidation={ identityMap }
```

Comment:
It feels like we may want to drop `extraValidation` or at least make it optional.